maze/README.rst

```diff
@@ -0,0 +1,78 @@
+Maze Solver Demo
+================
+Getting the D-Wave Quantum Computer to solve a maze!
+
+The purpose of this demo is to take a simple and familiar problem - solving a maze - and go through
+the steps of submitting the problem to the quantum computer.
+
+Code Overview
+-------------
+The idea is to describe the rules of how one can move through a maze with a set of constraints.
+Using the Ocean toolkit, these constraints can be converted into a binary quadratic model (BQM)
```

Comment:
The solution technique is to construct a set of constraints that enforces the rules of moving through a maze. These constraints are then converted by Ocean software tools to a ...